helm/nuts-knooppunt/README.md

```diff
@@ -0,0 +1,190 @@
+```
+
+## Configuration
+
+### Key Configuration Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `nuts.enabled` | Enable NUTS Node deployment | `true` |
+| `nuts.database.storage.size` | Storage size for NUTS database | `5Gi` |
+| `fhir.enabled` | Enable HAPI FHIR server | `false` |
+| `fhir.database.storage.size` | Storage size for FHIR database | `5Gi` |
+| `ingress.enabled` | Enable ingress | `true` |
+| `ingress.hosts` | Ingress hostnames | `[{host: chart-example.local}]` |
+| `replicaCount` | Number of replicas | `1` |
+
+### Database Storage Configuration
+
+The PostgreSQL database storage sizes are now configurable:
+
+```yaml
+# NUTS database storage
+nuts:
+  database:
+    storage:
+      size: 10Gi  # Adjust as needed
+
+# FHIR database storage
+fhir:
+  enabled: true
+  database:
+    storage:
+      size: 20Gi  # Adjust as needed
+```
+
+### Database Connection Configuration
+
+The NUTS node requires a PostgreSQL database connection. The database is automatically created by CloudNativePG, but you need to configure the connection string.
+
+#### Get the Database Connection String
+
+```bash
+# Get the auto-generated connection string from the CloudNativePG secret
+RELEASE_NAME=my-nuts-knooppunt
+kubectl get secret ${RELEASE_NAME}-nuts-db-app -o jsonpath='{.data.uri}' | base64 -d
+```
+
+#### Configure the Connection
+
+Create a `values.yaml` file with the connection string:
+
+```yaml
+# Enable FHIR server
+fhir:
+  enabled: true
+  database:
+    storage:
+      size: 10Gi
+
+# Configure NUTS node
+nuts:
+  enabled: true
+  database:
+    storage:
+      size: 10Gi
+  nuts:
+    config:
+      storage:
+        sql:
+          # Replace with the URI from the secret above
+          # Format: postgres://app:{password}@{release-name}-nuts-db-rw:5432/app
+          connection: "postgres://app:YOUR_PASSWORD_HERE@my-nuts-knooppunt-nuts-db-rw:5432/app"
+      strictmode: true
+      verbosity: info
+
+# Configure ingress
+ingress:
+  enabled: true
+  className: nginx
+  hosts:
+    - host: nuts.example.com
+      paths:
+        - path: /
+          pathType: Prefix
+```
+
+Then install or upgrade:
+
+```bash
+helm install my-nuts-knooppunt . -f values.yaml
+# or
+helm upgrade my-nuts-knooppunt . -f values.yaml
+```
+
+**Note**: The database password is auto-generated during installation. You must retrieve it from the secret after the first install and then upgrade the release with the connection string configured.
+
+## Uninstallation
+
+```bash
+helm uninstall my-nuts-knooppunt
+```
+
+Note: Persistent Volume Claims (PVCs) are retained by default. To delete them:
+
+```bash
+kubectl delete pvc -l app.kubernetes.io/instance=my-nuts-knooppunt
+```
+
+## Dependencies
+
+This chart depends on:
+
+- **fhir** (v0.1.0): HAPI FHIR server chart (file://../fhir)
+- **nuts-node-chart** (v0.0.4): NUTS Node official chart (https://nuts-foundation.github.io/nuts-node/)
```

Comment:
```suggestion
- **nuts-node-chart** (v0.0.5): NUTS Node official chart (https://nuts-foundation.github.io/nuts-node/)
```

Reply:
Set to 0.0.6 as a new chart is released.